fluid/object_detection/README.md

```diff
@@ -1,20 +1,27 @@
-The minimum PaddlePaddle version needed for the code sample in this directory is the lastest develop branch. If you are on a version of PaddlePaddle earlier than this, [please update your installation](http://www.paddlepaddle.org/docs/develop/documentation/en/build_and_install/pip_install_en.html).
+The minimum PaddlePaddle version needed for the code sample in this directory is the latest develop branch. If you are on a version of PaddlePaddle earlier than this, [please update your installation](http://www.paddlepaddle.org/docs/develop/documentation/en/build_and_install/pip_install_en.html).
 
 ---
 
 ## SSD Object Detection
```

Comment:
Add `Table of Contents` before line 5, refer https://github.com/PaddlePaddle/models/pull/979/files#diff-e74a88ec056840ea1a6d958b791284afR5

Reply:
revised